docs/file-conventions/-client.md

```diff
@@ -22,6 +22,7 @@ console.log(supportsVibrationAPI);
 // client: true | false
 ```
 
-See [Route Module][routemodule] for more information.
+See [Route Module][route-module] for more information.
 
-[routemodule]: ../route/route-module
+<!-- page doesn't exist anymore -->
+[route-module]: ../route/route-module
```

Comment:
This page (https://remix.run/docs/en/dev/route/route-module) doesn't seem to exist anymore

Reply:
For now, let's just remove the link and say like `please refer to the Route Module section in the sidebar`.  We'll have to figure out if we want index pages for some of those sidebar headings - Styling also has a similar challenge.